src/Hamiltonians/HubbardReal1D.jl

```diff
@@ -42,6 +43,15 @@ end
 dimension(::HubbardReal1D, address) = number_conserving_dimension(address)
 
 LOStructure(::Type{<:HubbardReal1D{<:Real}}) = IsHermitian()
+function LOStructure(::Type{<:HubbardReal1D{<:Complex,<:Any,U,T}}) where {U,T}
+    if iszero(T)
+        return IsDiagonal() # TODO: implement adjoint
+    elseif iszero(imag(U))
+        return IsHermitian() # still Hermitian with complex t
+    else
+        return AdjointUnknown() # diagonal elements are complex; TODO: implement adjoint
+    end
+end
```

Comment:
```suggestion
function LOStructure(::Type{<:HubbardReal1D{<:Complex,<:Any,U,T}}) where {U,T}
    if iszero(imag(U))
        return IsHermitian() # still Hermitian with complex t
    else
        return AdjointKnown()
    end
end
LinearAlgebra.adjoint(h::HubbardReal1D{TT,A,U,T}) where {TT<:Complex,A,U,T} = HubbardReal1D{TT,A,conj(U),conj(T)}(h.add)
```
Similarly I suggest to implement the adjoint for the other 1D Hubbard models with complex parameters.